publisher/proto/publisher.proto

```diff
@@ -16,7 +16,7 @@ message Request {
   bytes der = 1;
   string LogURL = 2;
   string LogPublicKey = 3;
-  bool precert = 4; // Deprecated, use kind instead
+  reserved 4;
```

Comment:
```suggestion
  reserved 4; // Previously precert
```

You won't need to regenerate the protobufs since this is just adding a comment.

Reply:
Interestingly, comments usually *do* require the files to be regenerated, because protoc includes comments in the generated code. But this *particular* comment, because it's on a no-op line, doesn't require a regen.

Reply:
Good to know about that. When I tested locally it was only this particular case.